### PR TITLE
UCM/CUDA/GTEST: pass cudaFree memory size to VM_UNMAP event

### PR DESCRIPTION
**Pull request by bureddy**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7272/ for details (Mellanox internal link).

**Comment by bureddy** [comment]
@yosefe  please review. 
this should go to v1.4.x too.

File: src/ucm/event/event.c [hunk 1]
```diff
@@ -452,13 +452,22 @@ ucm_dispatch_mem_type_free(void *addr, size_t length, ucm_mem_type_t mem_type)
...
+    CUdeviceptr pbase;
+    size_t psize;
 
     ucm_event_enter();
 
-    ucm_trace("ucm_cuMemFree(dptr=%p)",(void *)dptr);
+    ret = cuMemGetAddressRange(&pbase, &psize, (CUdeviceptr) dptr);
+    if (ret != CUDA_SUCCESS) {
+        ucm_warn("cuMemGetAddressRange(devPtr=%p) failed", (void *)dptr);
+        psize = 4; /* set minimum length */
```
[the remark this answers is not in the document]
Comment (bureddy) [reply src/ucm/event/event.c]: Just choose 4 to make it a some minimum range of addr?

[the remark this answers is not in the document]
Comment (yosefe) [reply src/ucm/event/event.c]: maybe 1?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7274/ for details (Mellanox internal link).

**Pull request merged**